apps/timetable/admin/index.html

```diff
@@ -135,9 +135,16 @@
         <!-- Batch edit template -->
         <script id="gh-batch-edit-template" type="text/template">
             <%= _.partial('admin-batch-edit', {
-                    'gh': gh,
-                    'data': data
-                }) %>
+                'gh': gh,
+                'data': data
+            }) %>
+        </script>
+
+        <script id="gh-batch-edit-event-row-template" type="text/template">
```

Comment:
Can you just add a comment above the script tag to be consistent with the other templates?